crates/bevy_render/src/texture/dds.rs

```diff
@@ -45,7 +45,12 @@ pub fn dds_buffer_to_image(
         depth_or_array_layers,
     }
     .physical_size(texture_format);
-    image.texture_descriptor.mip_level_count = dds.get_num_mipmap_levels();
+    let mut mip_map_level = dds.get_num_mipmap_levels();
+    if mip_map_level == 0 {
+        warn!("Mipmap levels for texture are 0, bumping them to 1");
+        mip_map_level = 1;
+    }
+    image.texture_descriptor.mip_level_count = mip_map_level;
```

Comment:
This could be refactored into a `match` statement:
```suggestion image.texture_descriptor.mip_level_count = match dds.get_num_mipmap_levels()  {
        0 => {
            warn!("Mipmap levels for texture are 0, bumping them to 1");
            
            1
        }
        level => level,
}
```